src/py/flwr/client/__init__.py

```diff
@@ -15,20 +15,16 @@
 """Flower client."""
 
 
-from .app import ClientLike as ClientLike
 from .app import run_client as run_client
 from .app import start_client as start_client
 from .app import start_numpy_client as start_numpy_client
-from .app import to_client as to_client
 from .client import Client as Client
 from .numpy_client import NumPyClient as NumPyClient
 
 __all__ = [
     "Client",
-    "ClientLike",
     "NumPyClient",
     "run_client",
     "start_client",
     "start_numpy_client",
-    "to_client",
```

Comment:
Can we continue to export these from `flwr.client`, and also include `ClientFn`, just like we do in `flwr.common` and `flwr.common.typing`?

Reply:
done